test/Microsoft.ML.Tests/CollectionDataSourceTests.cs

```diff
@@ -309,7 +309,6 @@ public class ConversionNullalbeClass
             public ulong? fuLong;
             public float? fFloat;
             public double? fDouble;
-            public bool? fBool;
```

Comment:
>public bool? fBool; [](start = 12, length = 19)

This kinda make me sad.
I understand what we want to have pure framework, and as workaround we force people to cast everything to float which support NaNs, it just in my life I worked with Something2SQL frameworks, and quite often you have nullable boolean columns, and if they fetch through DTO object, it will make that object uncastable /unwrappable for our collection to dataview.
Which will lead to bunch of painful casting/objects recreations.
But again, what do I know about life.

Reply:
yep, read Tom's comment on the issue page you might become happy! :)

---
In reply to: [211109825](https://github.com/dotnet/machinelearning/pull/692#discussion_r211109825) [](ancestors = 211109825)